code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -79,8 +108,15 @@
 					
 /mob/living/simple_animal/mouse/Move()
 	. = ..()
-	if(!stat)
+	if(!stat && !key)
 		eat_cheese()
+	if(key && stat == CONSCIOUS)
```

Comment:
this and the above are saying `stat == CONSCIOUS` in two different ways

Reply:
Changed the check to be more consistant